docs/book/src/developer/tilt.md

```diff
@@ -54,30 +92,14 @@ extra_args:
   ibmcloud:
     - '-v=5'
 ```
+---
+## Different flavors of deploying workload clusters using CAPIBM.
 
 ### 1.  Configuration to deploy PowerVS workload cluster with external cloud controller manager
 
-To deploy workload cluster with [PowerVS cloud controller manager](/topics/powervs/external-cloud-provider.html) which is currently in experimental stage, Set `PROVIDER_ID_FORMAT` to `v2` and enable cluster resourceset feature gate under kustomize_substitutions.
-Currently, [ClusterResourceset](https://cluster-api.sigs.k8s.io/tasks/experimental-features/cluster-resource-set.html) is experimental feature so we need to enable the feature gate by setting `EXP_CLUSTER_RESOURCE_SET` variable under kustomize_substitutions.
+To deploy workload cluster with [PowerVS cloud controller manager](/topics/powervs/external-cloud-provider.html)(experimental) or to deploy workload cluster with [cloud controller manager](/topics/vpc/load-balancer.html)(experimental), set `PROVIDER_ID_FORMAT` to `v2` and enable cluster resourceset feature gate under kustomize_substitutions.
 
-```yaml
-default_registry: "gcr.io/you-project-name-here"
-provider_repos:
-- ../cluster-api-provider-ibmcloud
-enable_providers:
-- ibmcloud
-- kubeadm-bootstrap
-- kubeadm-control-plane
-kustomize_substitutions:
-  IBMCLOUD_API_KEY: "XXXXXXXXXXXXXXXXXX"
-  PROVIDER_ID_FORMAT: "v2"
-  EXP_CLUSTER_RESOURCE_SET: "true"
-```
-
-### 2.  Configuration to deploy VPC workload cluster with external cloud controller manager
-
-To deploy workload cluster with [cloud controller manager](/topics/vpc/load-balancer.html) which is currently in experimental stage, Set `PROVIDER_ID_FORMAT` to `v2` and enable cluster resourceset feature gate under kustomize_substitutions.
-Currently, [ClusterResourceset](https://cluster-api.sigs.k8s.io/tasks/experimental-features/cluster-resource-set.html) is experimental feature so we need to enable the feature gate by setting `EXP_CLUSTER_RESOURCE_SET` variable under kustomize_substitutions.
+Currently, [ClusterResourceSet](https://cluster-api.sigs.k8s.io/tasks/experimental-features/cluster-resource-set.html) is an experimental feature. This requires setting the feature gate `EXP_CLUSTER_RESOURCE_SET` to `true` under kustomize_substitutions.
```

Comment:
I see that is is being repeated in two sections here, maybe we can mention this point as a note in the beginning to avoid redundancy?